### ISSUE TITLE
About the setting.

### ISSUE THREAD

**Issue by kxgong** [VITA-Group/Nasty-Teacher#15]
Hi there, thanks for your great work!!!
I have a question about the setting of this work. Does this work assume that the 'stealer' can access and only access the logits output of the teacher model? And the model resources like model structure and parameters are invisible?

**Comment by HowieMa** [comment]
Hi, thanks for your interest in our work. Your understanding is correct. We assume the teacher model is a black-box API and we can only access its input-output behavior.

**Comment by kxgong** [comment]
Thanks for your instant reply! 
Nasty teacher is a solid work!
